docs/content/dagster-plus/managing-deployments/branch-deployments/change-tracking.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: "Tracking asset changes in Dagster+ Branch Deployments | Dagster Docs"
+description: "Change tracking in Branch Deployments makes it eaiser for you and your team to identify how changes in a pull request will impact data assets."
+---
+
+# Tracking asset changes in Dagster+ Branch Deployments
+
+<ApplicableDagsterProduct />
+
+Change tracking in Branch Deployments makes it eaiser for you and your team to identify how changes in a pull request will impact data assets. By the end of this guide, you'll understand how change tracking works and what types of asset changes can be detected.
```

Comment:
formatting question mostly for my own knowledge and the blog post i'm writing. We're calling this feature Change Tracking, so does that mean it should be capitalized, or does it stay capitalized according to the context it's used in (Change tracking for the start of a sentence; change tracking in the middle etc)

Reply:
The latter. It's the same for other features too, like Branch Deployments:

- **Referring to the feature itself** uses the proper noun: `The Branch Deployments feature allows you to...`
- **Referring to an instance of the feature** doesn't require a proper noun, so no caps: `To access a branch deployment in Dagster+...`

Does that clarify things?

Reply:
yes - thank you!